web/portal/platform/src/components/ActiveCalls/ActiveCalls.helpers.tsx

```diff
@@ -0,0 +1,42 @@
+import { CriteriaFilterValues } from '@irontec/ivoz-ui/components/List/Filter/ContentFilterDialog';
+
+import { WsFilters } from './types';
+
+export function queryStringCriteriaToFilters(
+  queryStringCriteria: CriteriaFilterValues
+): WsFilters {
+  if (!queryStringCriteria.length) {
+    return null;
+  }
+
+  const resp: WsFilters = {
+    trunks: {},
+  };
+
+  for (const idx in queryStringCriteria) {
+    const filter = queryStringCriteria[idx];
+    switch (filter.name) {
+      case 'Brand':
+        resp.trunks.b = filter.value as string | number;
+        break;
+      case 'Company':
+        resp.trunks.c = filter.value as string | number;
+        break;
+      case 'Carrier':
+        resp.trunks.cr = filter.value as string | number;
+        break;
+      case 'Direction':
+        if (filter.value === 'inbound') {
+          resp.trunks.dp = '*';
+        } else if (filter.value === 'outbound') {
+          resp.trunks.cr = '*';
+        }
+        break;
```

Comment:
Direction in this switch may lead to strange filtering. It's designed to set `dp` or `cr` to `*`, but only if there is no `dp` or `cr` value set.

If you filter Carrier = 1 and Direction = outboud (in that order), the value should be `cr1`, but it will be `*` because Direction filtering will be processed after Carrier one.